internal/middlewares/requestedRegions.go

```diff
@@ -10,57 +10,58 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func CreateDisplayPreferences(c *gin.Context, input *models.DisplayPreferencesInput) {
+func CreateRequestedRegions(c *gin.Context, input *models.RequestedRegionsInput) {
 	if err := c.ShouldBindJSON(&input); err != nil {
 		log.Error(err)
 		httpStatus, response := helpers.ErrorToJson(http.StatusBadRequest, err.Error())
 		c.JSON(httpStatus, response)
 		return
 	}
 
-	displayPreferences := hydrateDisplayPreferences(input)
-	if err := database.Db.Create(&displayPreferences).Error; err != nil {
+	requestedRegions := hydrateRequestedRegions(input)
```

Comment:
tiny detail but i don't think it is called hydrated if the object (RequestedRegions) hasn't been instantiated before